src/Native/System.Native/pal_time.h

```diff
@@ -5,6 +5,12 @@
 
 #include "pal_types.h"
 
+enum TimeConversionConstants
+{
+    SecondsToMicroSeconds = 1000000,  // 10^6
+    SecondsToNanoSeconds = 1000000000 // 10^9
+};
```

Comment:
Does this need to be in the .h? Also, I'd leave the enum unnamed if it's just to declare integer constants.